s3cmd

```diff
@@ -2346,6 +2375,22 @@ def cmd_setpolicy(args):
         output(u"%s: Policy updated" % uri)
     return EX_OK
 
+def cmd_listpolicy(args):
+    cfg = Config()
+    s3 = S3(cfg)
+    uri = S3Uri(args[0])
+
+    if len(args) != 1:
```

Comment:
Once you will fix the argc value in the command list, you don't need this test anymore as it is aleady done before starting the command.